### PR TITLE
Init point cloud and video augmentation module

### PR DESCRIPTION
**Pull request by shijianjian**
#### Changes

Fixes # (issue)

#### Type of change

- [ ] 📚  Documentation Update
- [ ] 🧪 Tests Cases
- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔬 New feature (non-breaking change which adds functionality)
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 This change requires a documentation update

#### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Did you update CHANGELOG in case of a major change?

### CODE REVIEW

**Review by edgarriba — COMMENTED** [review]
@shijianjian shouldnt we define a Pointcloud data structure too ? Do you have something in mind whether to represent dense or sparse pointcloud and how to deal with the memory footprint for the dense case ?

File: kornia/augmentation/pc/base.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+            return input, None, None, None
+        if C < 6:
+            return input[..., :3], None, None, input[..., 3:]
+        if C == 6:
+            return input[..., :3], input[..., 3:], None, None
+        if C < 9:
+            return input[..., :3], input[..., 3:6], None, input[..., 6:]
+        if C == 9:
+            return input[..., :3], input[..., 3:6], input[..., 6:], None
+        return input[..., :3], input[..., 3:6], input[..., 6:9], input[9:]
```
Comment (shijianjian) [inline kornia/augmentation/pc/base.py:62]: @edgarriba The fields for a point cloud object can be represented like above.

Also, I think we should support `.from_open3d` and `.to_opend3d`.
http://www.open3d.org/docs/release/python_api/open3d.geometry.PointCloud.html

Comment (edgarriba) [reply kornia/augmentation/pc/base.py:62]: 100% agree

**Comment by cjpurackal** [comment]
> @shijianjian shouldnt we define a Pointcloud data structure too ? Do you have something in mind whether to represent dense or sparse pointcloud and how to deal with the memory footprint for the dense case ?

This sounds good. I've been trying out some RGBD stuff and was wondering the same. Perhaps we can open a different discussion thread(issue) for this.

**Comment by shijianjian** [comment]
> > @shijianjian shouldnt we define a Pointcloud data structure too ? Do you have something in mind whether to represent dense or sparse pointcloud and how to deal with the memory footprint for the dense case ?
> 
> This sounds good. I've been trying out some RGBD stuff and was wondering the same. Perhaps we can open a different discussion thread(issue) for this.

@edgarriba @cjpurackal Can you point me some links as references? I am not sure what to do on this.

Probably worth having RGBD data structure.

**Comment by edgarriba** [comment]
@cjpurackal can you start that thread ?  Very often RGBD are just 4xHxW  but depending how we want to target pointclouds it can be defined as a just as sparse vectors

**Comment by cjpurackal** [comment]
> @cjpurackal can you start that thread ? Very often RGBD are just 4xHxW but depending how we want to target pointclouds it can be defined as a just as sparse vectors

will do

**Comment by johnnv1** [comment]
as part of https://github.com/kornia/kornia/discussions/2719 plan, we have deleted the `master` in favor of `main`, if anyone has an interest in keep/restart working on this, let me know to help you.

**Pull request closed without merging**